ouroboros-consensus-test/src/Test/LedgerTables.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+module Test.LedgerTables (
+    prop_stowable_laws
+  , prop_tablestuff_laws
+  ) where
+
+import           Data.Function (on)
+
+import           Ouroboros.Consensus.Ledger.Basics
+import           Ouroboros.Consensus.Ledger.Tables
+
+import           Test.QuickCheck
+
+-- | We will compare the Ledger Tables of the result because the comparison for
+-- the rest of the LedgerState takes considerably more time to run.
```

Comment:
```suggestion
-- | We compare the Ledger Tables of the result because the comparison with
-- the rest of the LedgerState takes considerably more time to run.
```